### ISSUE TITLE
new rule - ban emoji with skin tone modifiers

### ISSUE THREAD

**Issue by DetachHead** [rvben/rumdl#834]
### Problem or motivation

emojis with skin tone modifiers often don't render correctly depending on what platform the reader is using.

i would also argue skin tone is very rarely relevant to whatever your documentation is about lol. in most cases when it's used in markdown, it was probably added by accident due to the configuration of the author's emoji picker.

### Proposed solution

a rule to ban emojis with skin tone modifiers

### Alternatives considered

an option to specify the maximum supported unicode version, although i'm not very familiar with how emoji versioning works and i suspect this isn't as staightforward (it seems similar to JS features where compatibility needs to be determined for each individual emoji rather than the spec version)

### Feature category

New lint rule

### Example markdown

```markdown
<!-- error: skin tone modifiers are not allowed -->
👪🏻
```

### Contribution

- [ ] I would be willing to submit a PR to implement this feature
